src/tools/genpolicy/tests/main.rs

```diff
@@ -135,6 +135,11 @@ mod tests {
         runtests::<CreateSandboxRequest>("createsandbox").await;
     }
 
+    #[tokio::test]
+    async fn test_update_routes() {
+        runtests::<UpdateRoutesRequest>("updateroutes").await;
```

Comment:
Are we missing some test cases here?

Reply:
Whoops, forgot to check in that directory. See latest.